testing/acceptance_test.py

```diff
@@ -896,26 +897,41 @@ def test_has_plugin(self, request):
 class TestDurations:
     source = """
         import time
-        frag = 0.002
+        frag = 0.002             # 2 ms
         def test_something():
             pass
         def test_2():
-            time.sleep(frag*5)
+            time.sleep(frag*5)   # 10 ms: on windows might sleep < 0.05s
```

Comment:
Did you mean to write `0.005s` here (and below)? Otherwise I'm confused...

Reply:
yes sorry for the mistake